src/frontend/src/optimizer/plan_node/logical_delete.rs

```diff
@@ -101,10 +101,14 @@ impl ToBatch for LogicalDelete {
 
 impl ToStream for LogicalDelete {
     fn to_stream(&self) -> Result<PlanRef> {
-        unreachable!("delete should always be converted to batch plan");
+        Err(RwError::from(ErrorCode::InternalError(
+            "delete should always be converted to batch plan".to_string(),
+        )))
```

Comment:
I think this should be kept as it is, because it works like an assertion

Reply:
> If the latter,the panic is the strong contrains may be better.

What do you mean by "strong contrains"? `unreachable` is just a special kind of `panic`

Reply:
OK.If i am product user. And i run a sql and it panic. Is there some error to tell me what happened and what should i do next?

Reply:
Get